crates/server/src/main.rs

```diff
@@ -63,6 +63,11 @@ async fn main() {
             traces_sample_rate: 1.0,
             enable_profiling: true,
             profiles_sample_rate: 1.0,
+            environment: Some(
+                std::env::var("MASTIFF_ENVIRONMENT")
+                    .unwrap_or("development".into())
+                    .into(),
+            ),
```

Comment:
`sentry` uses `debug`/`release` to decide if it is `development`/`production`, but `rocksdb` is very slow in debug mode. Making it explicit here and defaulting to `development` unless `MASTIFF_ENVIRONMENT` is defined (which it is in production).